### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jgillson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Unobtrusive, inheritable-overridable-configurable, drop-in debug logging, that w
 * *separate logger, if needed*
 * *log method calls, also when exit scope*
 * *Prevents heavy computation of strings with `logger.debug { 'log me' }` block format.*
+* *ActiveSupport::Notifications integration for instrumenting/logging events on class and instance methods*
```
Comment (pboling) [inline README.md:33]:
```suggestion
* *ActiveSupport::Notifications integration for instrumenting/logging events on class and instance methods*
* *Optional instance variable logging*
```

Comment (pboling) [reply README.md:33]: Can you document the new support for instance variable logging also?

**Review by pboling — CHANGES_REQUESTED** [review]
Once more thing

File: README.md (hunk 1 above)
Comment (jgillson) [reply README.md:33]: Done 👍

**Review by pboling — APPROVED** [review]
LGTM

**Pull request merged**